bin/docserv-createconfig

```diff
@@ -267,6 +267,21 @@ def check_langs(langs, message, attempts = 3):
     sys.exit("Expected one or more of these:\n"
         "{}".format(", ".join(languages)))
 
+def check_productid(productid, message, attempts = 3):
+    productid_regex = r'(?P<productid>[-_.+a-zA-Z0-9]+)'
+    message_expected = ("Following chars accepted: -_.+a-zA-Z0-9")
+    while attempts:
+        if not productid:
+            productid = input(message)
+        match = re.match(productid_regex, productid)
+        if match and match.group('productid') == productid:
+            return productid
+        else:
+            attempts -= 1
+            print("Invalid Product ID '{}' given".format(productid))
+            print(message_expected)
+            productid = ""
+    sys.exit(1)
```

Comment:
I know this line literally came from me but maybe it is not completely ideal to exit without telling the user what happened... The following is nt particularly user-friendly but it gets the message across, I guess:

```suggestion
    sys.exit("Exiting. You entered invalid input too often.")
```

(Could you please reuse the same message that you decide on in this case for line 267/268 also, please?  My suggestion would be to move the current message on line 267/268 up, so that it is shown during the `if:` line 261 to 264.)